azkaban-common/src/main/java/azkaban/utils/FileIOUtils.java

```diff
@@ -84,9 +87,10 @@ public static String getSourcePathFromClass(Class<?> containedClass) {
   }
 
   /**
-   * Run a unix command that will symlink files, and recurse into directories.
+   * Run a unix command that will hard link files and recurse into directories.
```

Comment:
Could you add a comment that with this change, the project directory can't be stored in a different partition from the partition that the executions directory is in?

It is probably better to add it to the change description as well.